content/docs/gherkin/reference.md

```diff
@@ -99,7 +99,12 @@ You can write anything you like, as long as no line starts with a keyword.
 
 ## Rule
 
-The (optional) `Rule` keyword has been added in Gherkin v6. (Note that Gherkin 6 has not yet been incorporated into all implementation of Cucumber!)
+The (optional) `Rule` keyword has been part of Gherkin since v6. 
+
+{{% note "Cucumber Support for Rule"%}}
```

Comment:
I like that you added this note.
However, this might lead people to wonder whether thay can / cannot use the Rule keyword in their implementation / version; how would they find out? Can you add generic information on how they could find that out (without having to list them here; which we would have to maintain)

Reply:
You're right, this note raises more questions than answers.

I'm not sure a list would be so bad? What about, for each Cucumber implementation, we show either the version it landed in, or the GitHub issue for it. Then when all are done we can just update to something like "supported in all current versions, you may need to update if you're on an older one"

Reply:
Or could open an issue in the monorepo, track all the detail there, and update the docs when closing it - would that work?